client/src/views/AllocationView/AllocationView.tsx

```diff
@@ -129,6 +129,7 @@ const AllocationView = (): ReactElement => {
     refetchHistory();
     refetchWithdrawals();
     refetchProjectsDonors();
+    toastService.hideToast('confirmChanges');
```

Comment:
@jmikolajczyk: I tried to implement this hiding so that it still happens in `getShouldProjectBeAddedOrRemovedFromAllocation`, but triggering that via hooks in `AllocationView` proved to be complicated. I'm opened to different approaches if you have some.